src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -1197,6 +1199,21 @@ internal SourceFile(AssemblyInfo assembly, int id, DocumentHandle docHandle, Uri
             }
         }
 
+        private static string EscapeAscii(string path)
+        {
```

Comment:
If we do `path.Replace("\\", "/").Replace(":", "");` then this method can be simpler, no regex needed at all.